app/src/main/java/com/futurice/freesound/feature/home/DefaultUserDataModel.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2016 Futurice GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.futurice.freesound.feature.home;
+
+import com.futurice.freesound.network.api.FreeSoundApi;
+import com.futurice.freesound.network.api.model.UserResult;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.VisibleForTesting;
+
+import io.reactivex.Single;
+
+import static com.futurice.freesound.utils.Preconditions.get;
+
+final class DefaultUserDataModel implements UserDataModel {
```

Comment:
This `Default*` naming convention actually gets in the way here, because it *is* the default user, so to speak. Oh well.